projects/angular/src/forms/styles/_mixins.forms.scss

```diff
@@ -118,12 +118,16 @@
   cursor: not-allowed;
 }
 
-@mixin checkbox-radio-input-styles($size) {
+@mixin checkbox-radio-input-styles(
+  $size,
+  $input-top: clr-forms-radio-input-top,
+  $input-top-fallback: forms-variables.$clr-forms-radio-input-top
```

Comment:
What's the use of this variable if we use it only for the hidden input? If we are going to allow different top positioning of the checkbox then it should affect both the hidden and the visible checkbox.

Reply:
The input is opaque.
In it's initial state was hard coded to `top: variables.$clr_baselineRem_5px;` I decided to parameterize so the hidden can overlap with the visible checkbox with tokens.
The visible and the hidden checkbox cannot use same parameter unless we introduce some very complicated calculation for it.
I don't think there is much to gain from the top parameter.
@dtsanevmw I can revert my change.

Reply:
corrected in commit a7e1db6